src/brogue/Architect.c

```diff
@@ -34,11 +34,10 @@ boolean cellHasTerrainFlag(short x, short y, unsigned long flagMask) {
 #endif
 
 boolean checkLoopiness(short x, short y) {
-    boolean inString;
     short newX, newY, dir, sdir;
     short numStrings, maxStringLength, currentStringLength;
 
-    if (!(pmap[x][y].flags & IN_LOOP)) {
+    if (!(pmapAt((pos){x, y})->flags & IN_LOOP)) {
```

Comment:
Why this change? `pmap[x][y]` is used frequently in the codebase?

Reply:
Oh, this was unintentional - I was about to go down a different refactoring path before I noticed `distanceBetween`. I'll take it out of this PR

Reply:
Now removed from the PR